services/viva-ms/lambdas/syncExpiryTime.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-console */
+import to from 'await-to-js';
+import * as dynamoDb from '../../../libs/dynamoDb';
+
+import config from '../../../config';
+import params from '../../../libs/params';
+
+import { getFutureTimestamp, millisecondsToSeconds } from '../../../libs/timestampHelper';
+import {
+  DELETE_VIVA_CASE_AFTER_45_DAYS,
+  DELETE_VIVA_CASE_AFTER_72_HOURS,
+} from '../../../libs/constants';
+
+const VIVA_CASE_SSM_PARAMS = params.read(config.cases.providers.viva.envsKeyName);
+
+export async function main(event) {
+  const { caseKeys } = event.detail;
+
+  const [getCaseError, getCaseResponse] = await to(getCase(caseKeys));
+  if (getCaseError) {
+    throw getCaseError;
+  }
+
+  const caseItem = getCaseResponse.Items[0];
+
+  const vivaCaseSSMParams = await VIVA_CASE_SSM_PARAMS;
+  if (vivaCaseSSMParams.recurringFormId !== caseItem.currentFormId) {
+    console.info('(Viva-ms: syncExpiryTime): case is not an recurring form. Nothing to update.');
+    return true;
+  }
```

Comment:
Suggestion for change, this if statement could be used when getting an item from the DynamoDB response.

const caseItem = getCaseResponse.Items.find(item => item.currentFormId === vivaCaseSSMParams.recurringFormId)

if (!caseItem) {
   console.info('(Viva-ms: syncExpiryTime): case is not an recurring form. Nothing to update.');
   return true;
}

Reply:
hmm nja